migration/idempotent/007-tracing-tags.sql

```diff
@@ -98,7 +98,10 @@ $do$;
 
 DO $do$
 BEGIN
-	CREATE OPERATOR ps_trace.= (
+    -- We need to drop both the operator and the operator class if they exist
+    DROP OPERATOR CLASS IF EXISTS btree_tag_v_ops USING btree;
```

Comment:
I would suggest that you move both of these DROP statements to a new incremental migration, otherwise you're going to need to continue to implement incremental migration logic in this block.